MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,579 @@
+---
+layout: post
+title: Working with DataForm in Maui DataForm control | Syncfusion
+description: Learn here all about Working with DataForm support in Syncfusion Maui DataForm (SfDataForm) control and more.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
+
+## Auto-generating DataFormItems for the data field
+
+By default, the [DataFormItems]`Link` will be generated based on the property type. For example, the [DataFormTextItem]`Link` will be created for the `string` type property. 
```

Comment:
[DataFormTextItem]`Link` => [DataFormTextItem]

Remove link term in all places will add links later